### DISCUSSION TITLE
Action in mobile

### DISCUSSION THREAD

**Discussion by Airuop** [community/community#64372] [Actions] [answered]
Hi 
How we can start action manually ?
I can't find this button on GitHub mobile version

**Comment by Akash1134** [comment]
Hi @Airuop ,

Please check out [this changelog post](https://github.blog/changelog/2023-05-09-introducing-actions-on-the-repository-view-on-github-mobile/) and let us know if that helps.

Thanks!

**Comment by arafajs** [comment, marked as answer]
Currently, you cannot start an action manually on the GitHub mobile version. The "Run workflow" button is only available on the GitHub web version.

To start an action manually on GitHub web, follow these steps:

- Go to the repository that contains the workflow you want to run.
- Click on the Actions tab.
- In the left sidebar, click on the name of the workflow you want to run.
- Above the list of workflow runs, click on the Run workflow button.

**Reply by Airuop** [reply]
Right I know!
In the next update, please add the ability to run the workflow from the mobile app.

**Comment by 58juspa-58** [comment]
am not able to link my account once again t my whatsap

**Comment by mdtanvirahamedshanto** [comment]
Responce

**Comment by mdtanvirahamedshanto** [comment]
On GitHub.com, navigate to the main page of the repository. Under your repository name, click Actions. In the left sidebar, click the name of the workflow you want to run. Above the list of workflow runs, click the Run workflow button.
